packages/salesforcedx-vscode/README.md

```diff
@@ -21,7 +23,7 @@ To use Salesforce Extensions for VS Code, install all the extensions in this ext
 * [salesforcedx-vscode-apex](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex)  
    This extension uses the Apex Language Server to provide features such as syntax highlighting and code completion.
 * [salesforcedx-vscode-apex-debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger)  
-   This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs.
+   This extension enables VS Code to use the real-time Apex Debugger with your scratch orgs or to use ISV Customer Debugger.
```

Comment:
Same comment as above - but now I'm going to ask if ISV cust debugger can be used with any org?

Reply:
Just would say if you want to say here ...for ISV Customer Debugger for your subscribers' orgs. But I will defer to your discretion.